predicators/envs/burger.py

```diff
@@ -104,6 +109,12 @@ def __init__(self, use_gui: bool = True) -> None:
                              self._On_holds)
         self._OnNothing = Predicate("OnNothing", [self._object_type],
                                     self._OnNothing_holds)
+        # When an argument should be an item or a station but not a robot,
+        # having the argument more narrowly scoped than the object type would be
+        # helpful. But this is causing errors so we'll avoid it for now.
+        # # self._Clear = Predicate("Clear", [self._item_or_station_type],
+        #                         self._Clear_holds)
+        # TODO(ashay): fix the errors associated with this.
```

Comment:
if this isn't a TODO anymore; remove the comment

Reply:
short answer: this is still a TODO, but it is not a high priority since we don't seem to have any problems right now the way it is. 

longer answer:
Ok, so I'll expand on why I was trying to make a new type that was an "item or a station". Like, I wanted the type hierarchy to be `_object_type` -> `_item_or_station_type`-> <`_item_type`, `_station_type`>. 

Basically, for some of the `GoalHack` predicates, they take in two arguments. I initially made the first argument `_object_type`, so e.g. one of the GoalHacks was true whenever <the patty was above ANY object + the patty was cooked> but then I was seeing this become true at times when I didn't want it to, because like, I saw `GoalHack(robot, patty)`. I temporarily dealt with this by just having 2 or more GoalHacks for the patty being on the cutting board + cooked, and another for the patty being on the grill + cooked., and another for being on top of the bottom bun + cooked, but it would be cleaner to have fewer goalhack predicates. 

But, I came across some weird error when trying to use `_item_or_station_type`. It's low priority so I think we should come back to it later, but I want to leave it as a comment since when we get to the final experiments for the paper, for cleanliness, we probably as few GoalHack predicates as possible.

Reply:
sounds good! We can revisit this after we start running experiments + getting results in Burger (which will hopefully be v soon!)